src/components/SelectEstimatedFee.vue

```diff
@@ -0,0 +1,140 @@
+<template>
+  <div v-if="estimationOptions" class="estimation-table">
```

Comment:
Could we move this `if` condition to the parent ([SetValueTransferFee.vue](https://github.com/witnet/sheikah/pull/1801/files#diff-0b5cb4cab008d30d8674a21ff613be33644240d94a8aee525473ea863502beb4))? If we do that, we will render this component only when it's necessary